core/src/main/resources/reference.conf

```diff
@@ -2182,6 +2182,22 @@ datastax-java-driver {
     # Overridable in a profile: yes
     prepare-on-all-nodes = true
 
+    # There is known problem in CQL 4.x when prepared statement invalidation could be voided:
+    # https://github.com/scylladb/scylladb/issues/20860
+    # When it happens metadata on client side does not match data and deserialization can go wrong in many ways
+    # To avoid driver can disable skip metadata flag to make server respond with metadata on every query.
+    # Unfortunately it causes excessive network traffic and CPU overhead on both server and driver side.
+    # This option controls how driver resolves skip metadata flag for CQL4 prepared statements.
+    # "smart" - disable flag only for wildcard selects (select * from) and selects that
+    #     return UDTs, including collections of UDTs and maps that contain UDTs
+    # "always-on" - flag is always set
+    # "always-off" - flag is always disabled
```

Comment:
If flag is disabled, what does it mean? That we DO skip or do NOT skip metadata?

Reply:
I have reverted it back to true/false (completely forgot that prepared statement invalidation is completely broken, not only for whildcards selects and udts) and added more text.

Reply:
added more info